pyproject.toml

```diff
@@ -107,3 +104,8 @@ disable = "R0913"
 
 [tool.pylint.format]
 max-line-length = "120"
+
+
+[tool.ruff]
+src = ["src", "test"]
```

Comment:
```suggestion
src = ["src", "tests"]
```

Reply:
@jsstevenson Should this be resolved?

Reply:
This PR is still showing up in https://github.com/pulls/review-requested for me. See above comment